package.json

```diff
@@ -26,18 +26,28 @@
   },
   "devDependencies": {
     "condition-circle": "1.5.0",
+    "eslint": "^5.7.0",
     "github-post-release": "1.13.1",
     "husky": "^0.14.3",
+    "jest": "^23.6.0",
     "semantic-release": "8.2.0",
     "simple-commit-message": "3.3.1"
   },
   "scripts": {
     "precommit": "eslint *.js",
-    "semantic-release": "semantic-release pre && npm publish --access public && semantic-release post"
+    "semantic-release": "semantic-release pre && npm publish --access public && semantic-release post",
+    "start": "yarn run test:watch",
+    "test": "jest",
+    "test:watch": "jest --watchAll"
```

Comment:
Same as on the other one, this will need to be changed to `test-watch` to prevent a merge conflict/duplication.

Reply:
Rebased this PR on the updated other one so this fix should be included. Thanks again!